student-store-ui/src/components/App/App.jsx

```diff
@@ -6,27 +6,90 @@ import Home from "../Home/Home"
 import "./App.css"
 import Footer from "../Footer/Footer"
 import ProductDetail from "../ProductDetail/ProductDetail"
-import { useState } from "react"
-
-//handleOnToggle - func toggle open/closed state of sidebar
-//handleAddItemToCart - func, accept 1 argue: productId, add it to [shoppingCart], set quantity to 1, add price to total
-//handleRemoveItemFromCart - accept single: productId, decrease by 1, if new quantity is 0, remove from shopping car
-//handleOnCheckoutFormChange - func, two argue: name & value, update the checkoutForm
-//handleOnSubmitCheckoutForm - func, submit user's order to the API using the axios.post, lots of other stuff
+import { useState, useEffect} from "react"
+import axios from "axios"
 
 export default function App() {
   const [isOpen, setIsOpen] = useState(false);
-  const [shoppingCart, setShoppingCar] = useState("");
-  const [products, setProducts] = useState("");
+  const [shoppingCart, setShoppingCart] = useState({});
```

Comment:
nit: initializing with empty object seems weird. Maybe use null. Also maybe this is better as a Map 

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Map